components/annotation_class.py

```diff
@@ -0,0 +1,158 @@
+import dash_mantine_components as dmc
+from dash import html, dcc
+from dash_iconify import DashIconify
+
+
+# This function generates a class component with all its buttons (hide/show, edit, delete)
+# The class color is used as an ID to identify a class (as all class colors are unique and cannot be modified)
+def annotation_class_item(class_color, class_label):
```

Comment:
Could we add `cursor: 'pointer'` to the style for these items so that they are clearly clickable?

Reply:
Yes I added a hover as well as a curser and styled the border and background color a bit more.



https://github.com/mlexchange/mlex_highres_segmentation/assets/56569375/c33801de-bd29-4743-8462-68144905243e

Reply:
Love!